data/json/monsters/reptile_amphibian.json

```diff
@@ -284,9 +284,9 @@
     "morale": 100,
     "aggro_character": false,
     "melee_skill": 9,
-    "melee_dice": 8,
+    "melee_dice": 6,
     "melee_dice_sides": 8,
-    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
+    "melee_damage": [ { "damage_type": "cut", "amount": 12 } ],
```

Comment:
```suggestion
    "melee_damage": [ { "damage_type": "stab", "amount": 8 } ],
```

Reply:
These bites should probably be stab instead of cut so I've shrunk their damage output.